independent-projects/bootstrap/maven-resolver/pom.xml

```diff
@@ -131,7 +131,17 @@
                         </goals>
                     </execution>
                 </executions>
+            </plugin> -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <environmentVariables>
+                        <!-- to verify that MAVEN_OPTS filtering works in BootstrapMavenOptions -->
+                        <MAVEN_OPTS>--add-opens java.base/java.util=ALL-UNNAMED</MAVEN_OPTS>
```

Comment:
Something like https://junit-pioneer.org/docs/environment-variables/ could be used instead but that sounds rather "hacky".